libraries/tests/mbed/pwm/main.cpp

```diff
@@ -19,15 +19,15 @@ float value = 0.75;
 int main() {
 #if defined(TARGET_FF_ARDUINO)
     PwmOut pwm(D9);
+    int period_us = 40000; // allows to test various periods
```

Comment:
Hi - change from 10 to 40ms is my mistake. Was just easier to read 0.75 duty cycle of 40 ... I will move it back to 10.
And yes, idea is to change manually, in the same than other duty cycles than 0.75 can be tested.  A next step could be to add a dedicated test on period changes, but not sure how to automatically check the result.